examples/fluids/qfunctions/shocktube.h

```diff
@@ -287,7 +287,8 @@ CEED_QFUNCTION(EulerShockTube)(void *ctx, CeedInt Q,
       CeedScalar j_vec[3] = {0.0};        // unit vector aligned with the density gradient
       CeedScalar j_gradn[3] = {0.0};      // j * grad(N)
       CeedScalar h_shock = 0.0;           // element lengthscale
-      CeedScalar acoustic_vel = 0.0;      // characteristic velocity, set to acoustic speed
+      CeedScalar acoustic_vel =
+        0.0;      // characteristic velocity, set to acoustic speed
```

Comment:
`make style` sometimes does weird things and we probably need to fix some of them manually.
how about
```
CeedScalar acoustic_vel = 0.0;      
// characteristic velocity, set to acoustic speed
```

Reply:
This is what happened for the above comment on `j_gradn[j]`. I'll go through and clean these up.